test/aio5gc/msg/nas/builder/registrationAccept.go

```diff
@@ -61,7 +61,7 @@ func buildRegistrationAccept(ue *context.UEContext) (nasMsg *nas.Message, err er
 	registrationAccept.TAIList.SetLen(uint8(len(taiListNas)))
 	registrationAccept.TAIList.SetPartialTrackingAreaIdentityList(taiListNas)
 
-	nssai := ue.GetNssai()
+	nssai := ue.GetSecurityContext().GetDefaultSNssai()
```

Comment:
Be careful in mixing up too much SecurityContext (UDM/AUSF stuff) with data that is provisioned for the UE in UDR.